helm_deploy/whereabouts-api/templates/deployment.yaml

```diff
@@ -25,6 +26,15 @@ spec:
         app: {{ template "app.name" . }}
         release: {{ .Release.Name }}
     spec:
+      affinity:
+        podAntiAffinity:
+          preferredDuringSchedulingIgnoredDuringExecution:
```

Comment:
`preferred` since if we use `required` then it won't deploy the other pods if it can't find a node